Core/DISET/private/Transports/SSL/SocketInfoFactory.py

```diff
@@ -57,7 +57,7 @@ def __sockConnect( self, hostAddress, sockType, timeout, retries ):
     osSocket = socket.socket( sockType, socket.SOCK_STREAM )
     #osSocket.setblocking( 0 )
     if timeout:
-      osSocket.settimeout( 5 )
+      osSocket.settimeout( 0.5 ) # we retry 3 times, which is 3 sec + 5 times retry = 15 sec.
```

Comment:
15 sec is not true any more. When we have ipv6 enabled we have a big problem... That is why I decreased the timeout. What I discovered is we have a list of addresses (ipv6 and ip4). ipv6 always  timeout. So 3 times 5 is 15 sec. For example:
2015-10-21 15:51:54 UTC tt[MG1M]  DEBUG: Connecting to: dips://lbtestvobox2.cern.ch:9167/Bookkeeping/TestBookkeepingManager
2015-10-21 15:51:54 UTC tt[MG1M]  DEBUG: Using proxy /tmp/x509up_u29854
retries 2 ('2001:1458:301:bc::100:ec', 9167, 0, 0) 10
EX: timed out ('2001:1458:301:bc::100:ec', 9167, 0, 0)
0.500597953796
EX: timed out ('2001:1458:301:bc::100:ec', 9167, 0, 0)
0.500988006592
EX: timed out ('2001:1458:301:bc::100:ec', 9167, 0, 0)
0.501138925552
retries 2 ('128.142.140.250', 9167) 2
2015-10-21 15:51:55 UTC tt[MG1M]  DEBUG:  Authenticated peer (/DC=ch/DC=cern/OU=computers/CN=lbtestvobox2.cern.ch)
2015-10-21 15:51:55 UTC tt[MG1M]  DEBUG: New session connecting to server at ('lbtestvobox2.cern.ch', 9167)
2015-10-21 15:51:55 UTC tt[MG1M]  DEBUG: New connection -> 128.142.140.250:9167
2015-10-21 15:51:56 UTC tt[MG1M]  DEBUG: Closing socket
{'OK': True, 'Value': [1825423375817125]}
Elapsed time: 2.51006507874